addons/mail/static/src/components/video_room/video_room.js

```diff
@@ -0,0 +1,100 @@
+odoo.define('mail/static/src/components/video_room/video_room.js', function (require) {
+'use strict';
+
+
+const useStore = require('mail/static/src/component_hooks/use_store/use_store.js');
+const useRefs = require('mail/static/src/component_hooks/use_refs/use_refs.js');
+
+const { Component, useState } = owl;
+
+class VideoRoom extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const chatRoom = this.env.models['mail.chat_room'].get(props.roomLocalId);
+            const mailRtc = this.env.mailRtc;
+            return {
+                chatRoom: chatRoom ? chatRoom.__state : undefined,
+                partnerRoot: this.env.messaging.partnerRoot,
+                activePeers: mailRtc.activePeers,
+            };
+            this.localToken = '';
```

Comment:
I'm not super fond of the word "token": term's too broad, and to me `token` is something too technical. I prefer something like UUID. Also don't forget to clarify "of what" when talking about these UUID. For instance here, instead of just "local token/UUID" prefer something like `currentRtcRoomUUID`.